File: Java/Easy/1678.java
class Solution {
    public String interpret(String command) {
        String answer = "";
        for (int i = 0; i < command.length(); i++) {
            char c = command.charAt(i);
            if (c == 'G') {
                answer += 'G';
            } else if (c == 'a') {
                answer += "al";
                i += 2;
            } else if (c == ')') {
                answer += 'o';
            }
        }
        return answer;
    }
}

File: C/Easy/1678.c
char *interpret(char *command)
{
    char *answer;
    int count = 0;
    answer = malloc(101 * sizeof(char));
    int len = strlen(command);
    for (int i = 0; i < len; i++)
    {
        if (command[i] == 'G')
        { // G
            answer[count] = 'G';
            count++;
        }
        else if (command[i] == '(')
        {
            i++;
            if (command[i] == ')')
            { //()
                answer[count] = 'o';
                count++;
            }
            else
            { //(al)
                i++;
                i++;
                answer[count] = 'a';
                count++;
                answer[count] = 'l';
                count++;
            }
        }
    }

    answer[count] = '\0';
    return answer;
}